components/jobCard/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+
+import {
+  CheckCircleIcon,
+  ChevronRightIcon,
+  UserGroupIcon,
+} from "@heroicons/react/24/outline";
+import Image from "next/image";
+import PropTypes, { InferProps } from "prop-types";
+import Button from "components/atoms/Button/Button";
+
+const jobDataProps = {
+  datas: Array<any>,
+  alt: PropTypes.string,
+};
+
+const JobCard = ({ datas, alt }: InferProps<typeof jobDataProps>) => {
+  return (
+    <div>
+      {datas.flatMap((data: any) => {
```

Comment:
the looping of job offers should be performed on the page not on the JobCard component